sbc_compassion/models/correspondence_template.py

```diff
@@ -67,23 +67,27 @@ class CorrespondenceTemplate(models.Model):
         help='Y Position of the upper-right corner of the bluesquare '
              'in pixels')
     qrcode_x_min = fields.Integer(
+        compute="_compute_size",
```

Comment:
Is it useful to keep this field if the `qr_code` area will always be in the same place ? Is it useful to display the information in the view ? I feel that we can remove these 4 fields along with the `_compute_size` method.